topoSort/ReconBuilder.py

```diff
@@ -0,0 +1,272 @@
+# ReconBuilder.py
+# This code builds a representation of a host tree, parasite tree, and a reconfiguration
+# as specified in Recon.py using the host tree, parasite tree, and reconfiguration 
+# representation in the DTL.
+
+from Tree import NodeLayout
+from Tree import TreeType
+from TreeFormatConverter import dict_to_tree
+
+__all__ = ["build_trees_with_temporal_order", "build_temporal_graph", "topological_order"]
```

Comment:
I still think you should only export `build_trees_with_temporal_order`.

Reply:
I think all three functions are useful. I might use the latter two functions myself, so I think I'll keep it there for now.